doc/nrf/gs_installing.rst

```diff
@@ -462,6 +462,14 @@ Visual Studio Code extension
 
 |vsc_extension_instructions|
 
+.. vsc_mig_note_start
+
+.. note::
+
+   If you are building the application or sample using SEGGER Embedded Studio IDE or on the command line and want to migrate to |VSC| extension IDE, follow the instructions in the `migrating from other IDEs to VS Code <Migrating IDE_>`_ documentation.
```

Comment:
```suggestion
   If you are building the application or sample using SEGGER Embedded Studio IDE or on the command line and want to migrate to |VSC|, follow the instructions in the `migrating from other IDEs to VS Code <Migrating IDE_>`_ documentation.
```

Reply:
Or maybe:
```suggestion
   If you are building the application or sample using SEGGER Embedded Studio IDE or on the command line and want to migrate to |VSC|, follow the instructions for `migrating from other IDEs <Migrating IDE_>`_ in the extension's documentation.
```

Reply:
I would keep the 1st comment as we are referring to the "Migrating from other IDEs to VS Code" documentation here.